CHANGELOG.md

```diff
@@ -12,6 +12,32 @@ The format is inspired by [Keep a Changelog].
 
 <!-- TOWNCRIER -->
 
+## 21.3.10 (2022-02-23)
+
+| Protocol          | Version   |
+|:------------------|:---------:|
+| Consensus         | 4.0.0     |
+| Runtime Host      | 4.0.0     |
+| Runtime Committee | 3.0.0     |
+
+### Features
+
+- go/common/cbor: Bump decoding limits for RPC
+  ([#4478](https://github.com/oasisprotocol/oasis-core/issues/4478))
+
+### Bug Fixes
+
+- go/runtime/scheduling: Attempt to continue scheduling
+  ([#4453](https://github.com/oasisprotocol/oasis-core/issues/4453))
+
+### Internal Changes
+
+- runtime: Bump tendermint-rs to 0.23.1
```

Comment:
Lets edit this to 0.23.5, since the other bump was marked as trivial

Reply:
Fixed.